src/test/java/com/epam/tc/hw2/WebDriverElement.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.tc.hw2;
```

Comment:
Files from hw2 should not be in PR fro hw3

Reply:
it is ready